src/cli/cmds/help.ts

```diff
@@ -14,6 +14,57 @@
  *  limitations under the License.
  */
 
-export async function help(): Promise<void> {
+import Log from '../../lib/Log';
+import {ParsedArgs} from 'minimist';
+
+const HELP_MESSAGES = new Map<string, string>(
+    [
+      ['main', [
+        'llama-pack [command] <options>',
+        '',
+        '',
+        'build ............... generates an Android APK from a TWA Project',
+        'help ................ shows this menu',
+        'init ................ initializes a new TWA Project',
+        'update .............. updates an existing TWA Project with the latest llama-pack template',
+      ].join('\n')],
+      ['init', [
+        'Usage:',
+        '',
+        '',
+        'lama-pack init --manifest=[web-manifest-url]',
+        '',
+        '',
+        'Options:',
+        '--directory ......... path where to generate the project. Defaults to the current' +
+            ' directory',
+      ].join('\n')],
+      ['build', [
+        'Usage:',
+        '',
+        '',
+        'llama-pack build',
+      ].join('\n')],
+      ['update', [
+        'Usage:',
+        '',
+        '',
+        'llama-pack update',
+      ].join('\n')],
+    ],
+);
+
+export async function help(args: ParsedArgs, log = new Log('help')): Promise<void> {
+  // minimist uses an `_` object to store details.
+  const command = args._[1];
+  const message = HELP_MESSAGES.get(command) || HELP_MESSAGES.get('main');
+
+  // Should never happen, as we know we have a message for main,
+  // but TypeScript doesn't know it.
+  if (message === undefined) {
```

Comment:
Yeah, so what you want here is an `!` (a (Non-null assertion operator)[https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator]).

Reply:
Interestingly, the linter configuration dislikes it:

```
 69:12  warning  Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion
```

Reply:
Here's the rule: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-non-null-assertion.md

I can keep the current role and the rule our add an eslint-ignore-line. I kind of prefer the first.

Reply:
I disagree with that rule - the point of strict null checking is for the compiler to catch things for you that you would have missed. In this case, we've acknowledged the possibility of null and said that we want to crash if that's the case.

Sorry, I don't understand what you're saying with "I can keep the current role and the rule"

Reply:
I agree. Changed eslint.json to remove the rule and removed the unnecessary check from help.ts.